meteor-app/imports/api/parts/methods.js

```diff
@@ -6,21 +6,23 @@ const debug = require('debug')('b2b:parts')
 
 function calcRetail(price){
   if (price <= 6000) {
-    const retailPrice = (parseInt(price, 10) * 100) * 2
+    const retailPrice = parseInt(price * 100, 10) * 2
```

Comment:
If price is a string, it should be converted to an integer before you multiply it. The previous version of the code was correct

Reply:
The previous version of this code errors out and skips over 1000 parts while parsing the xlsx file , this way it doesn't throw and error and successfully inserts the prices

Reply:
But the logic is wrong. You probably need to put a try/catch around the calculation, and spit out an error message for each one that fails, because you want to know what doesn't work and why. If the reason is because the price is something like "$20/box" then you probably need to clean the string before converting it to a number.